org.mixedrealitytoolkit.input/Editor/MRTK.Input.Editor.asmdef

```diff
@@ -22,15 +22,10 @@
     "defineConstraints": [],
     "versionDefines": [
         {
-            "name": "com.atteneder.gltfast",
+            "name": "com.unity.cloud.gltfast",
             "expression": "",
             "define": "GLTFAST_PRESENT"
         },
-        {
-            "name": "com.atteneder.ktx",
```

Comment:
@keveleigh , should we still let people use "com.atteneder.gltfast" and "com.atteneder.ktx" if they wanted to?  Or should we remove completely?